packages/paste-core/components/inline-code/src/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface InlineCodeProps extends Partial<Omit<HTMLDivElement, 'children'>> {
```

Comment:
question: Why do we need a partial here?

Reply:
Not adding Partial here leads to the following error:
<img width="792" alt="image" src="https://user-images.githubusercontent.com/77117846/187004935-1c782ffd-ef88-443a-bfde-e4b4e8e23e4b.png">
This is something I got from a PR from @shleewhite, can you explain it a bit more Lee?

Reply:
Oh, I was getting a huge typescript error that I was missing like every HTML element attribute when I tried to use the component. Partial fixed the issue 🤷🏽 

I also had to omit children because HTMLDivElement's child prop is not React.ReactNode.